lib/rpc.ts

```diff
@@ -139,7 +143,7 @@ export class Rpc extends EventEmitter {
   public async postMessageForward(fwdDest: string, data: any): Promise<void> {
     const msg: IMsgCustom = {mtype: MsgType.Custom, data};
     if (fwdDest) { msg.mdest = fwdDest; }
-    await this._sendMessage(msg);
+    await this._callWrapper(() => this._sendMessage(msg));
```

Comment:
Are wrappers for sending messages actually needed? If so, you don't think they need to be distinguished from wrappers for method calls?

Reply:
In grist, wrapping sending message allows to `ping` the `inactivityTimer` before sending a message. Using the same wrapper as for function call happens to work. 
I cannot think of a scenario where distinguishing between both would be needed, but I can add it.

Reply:
No need to add it, but please document. E.g. the user would need to know that the `callFunc` for sending a message may return immediately even if the message was only queued.

Reply:
> No need to add it, but please document. E.g. the user would need to know that the callFunc for > sending a message may return immediately even if the message was only queued
I explained elsewhere, we can do without wrapping send message and this is going to be removed from the PR.